client/components/videos-ui/style.scss

```diff
@@ -354,6 +354,8 @@
 			margin: 0 auto;
 			max-width: 1160px;
 			padding: 0 20px 80px;
+			align-self: center;
```

Comment:
With width 100%, I wouldn't imagine this is necessary; feel free to remove and deploy if so. Regardless, thanks for the fix!

Reply:
yup, it's not necessary. Good catch 🎉